spec/features/authentication_spec.rb

```diff
@@ -1,28 +1,55 @@
 require 'rails_helper'
 
-describe "the signin process", :type => :feature do
-  let(:user) { User.create(:email => 'user@example.com', :password => 'password') }
+describe 'Authentication' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.